cms/static/sass/elements/_tooltip.scss

```diff
@@ -0,0 +1,27 @@
+.tooltip {
```

Comment:
No. tooltip was part of the base in studio. And I basically refactored the styles related to tooltip into elements. However, because of the different usage in pattern library, we needed a scss in element folder and another scss in element-v2 folder.